kolibri/core/auth/utils.py

```diff
@@ -107,7 +109,15 @@ def _merge_log_data(LogModel):
             log_map[log.id] = new_log.id
             if new_log.id not in target_log_ids:
                 new_logs.append(new_log)
-        LogModel.objects.bulk_create(new_logs, batch_size=750)
+
+        vendor = connections[LogModel.objects.db].vendor
+        batch_size = (
+            calculate_max_sqlite_variables() // len(LogModel._meta.fields)
+            if vendor == "sqlite"
+            else 750
+        )
```

Comment:
(I used an inline conditional because the linting was complaining about function complexity and I didn't have it in me to try to start refactoring other stuff :) )